README.md

```diff
@@ -109,6 +109,25 @@ See [action.yml](./action.yml) for a full list of supported arguments and their
     fail: true
 ```
 
+## Utilising the cache feature
+
+In order to mitigate issues regarding rate limiting or to reduce stress on external resources, one can setup lychee's cache similar to this:
+
+```yml
+- name: Restore lychee cache
+  uses: actions/cache@v3
+  with:
+    path: .lycheecache
+    key: cache-lychee-${{ github.sha }}
+    restore-keys: cache-lychee-
+
+- name: Run lychee
+  uses: lycheeverse/lychee-action@v1.5.0
+  with:
+    args: '--cache --max-cache-age 1d'
+    lycheeVersion: 0.11.0
```

Comment:
The PR is still a draft to gather first feedback.

v0.11.0 is also not released yet.
Once lychee-action v1.6.0 updates the used default version, the value doesn't need to get passed of course.

Reply:
Updated to [0.10.1](https://github.com/lycheeverse/lychee/releases/tag/v0.10.1) with the cache fix.

Do you want to merge it like that or wait until the default version for the action is bumped?

Reply:
0.10.1 is the default version now. 🎉 Can you update that section and I'll merge it right away?